verilog/tools/kythe/kythe_proto_output.cc

```diff
@@ -12,15 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "verilog/tools/kythe/kythe_proto_output.h"
-
-#include <unistd.h>
+#ifndef _WIN32
+#include <unistd.h>  // for STDOUT_FILENO
+#else
+#include <stdio.h>
+#define STDOUT_FILENO _fileno(stdout)
+#endif
 
 #include "google/protobuf/io/coded_stream.h"
 #include "google/protobuf/io/zero_copy_stream_impl.h"
 #include "third_party/proto/kythe/storage.pb.h"
 #include "verilog/tools/kythe/kythe_facts.h"
 #include "verilog/tools/kythe/kythe_facts_extractor.h"
+#include "verilog/tools/kythe/kythe_proto_output.h"
```

Comment:
We place the header of the particular implementation always at the top (because then it is easy to see if there are some hidden dependencies), so the original location around line 14 was correct.

Reply:
The formatting was [suggested by the CI server](https://github.com/google/verible/runs/2423970246#step:5:32), am I understanding something the wrong way?

Reply:
It is important to have it as the very first include, then the formatter recognizes it. I suppose it was shifting under the the include changes you made, so it was not recognized as such, and thus suggested to be sorted alphabetically with the rest.

So if you move it to the top as first include, it should work.